Adyen/Utilities/LocalizationParameters.swift

```diff
@@ -40,5 +43,18 @@ public struct LocalizationParameters: Equatable {
         self.tableName = tableName
         self.keySeparator = keySeparator
         self.locale = locale ?? Locale.current.identifier
+        self.enforcedLocale = nil
+    }
+
+    /// Initializes LocalizationParameters with enforced locale.
+    ///
+    /// - Parameters:
+    ///   - locale: The locale to be enforced.
+    public init(locale: String) {
```

Comment:
```suggestion
    public init(enforcedLocale: String) {
```

But: Why not add the enforced locale to the end of the already existing init with a default value of nil? so it is not a breaking change :)

Reply:
Because those are mutually exclusive.
Also there is no breaking change - we are introducing new init, old one remain untouched.

Reply:
in v6 we can turn this into enum and use "A" or "B" 🤔

Reply:
Fixed [e34629f](https://github.com/Adyen/adyen-ios/pull/1259/commits/e34629fcff05e6cb6e663884de456d76a8fd4fee)